src/services/activityReports.js

```diff
@@ -850,11 +850,16 @@ export async function activityReportAlerts(userId, {
         [Op.and]: scopes,
         [Op.or]: [
           {
-            [Op.or]: [
-              { calculatedStatus: REPORT_STATUSES.SUBMITTED },
-              { calculatedStatus: REPORT_STATUSES.NEEDS_ACTION },
-            ],
-            '$approvers.userId$': userId,
+            calculatedStatus: {
+              [Op.in]: [
+                REPORT_STATUSES.SUBMITTED,
+                REPORT_STATUSES.NEEDS_ACTION,
+              ],
+            },
+            id: {
```

Comment:
the changes to this file were necessary as it seems there was name collision happening with the other literals and the joined table aliases